ods_ci/tests/Tests/400__ods_dashboard/415__ods_dashboard_projects/415__ods_dashboard_projects_additional.robot

```diff
@@ -4,6 +4,7 @@ Resource           ../../../Resources/OCP.resource
 Resource           ../../../Resources/ODS.robot
 Resource           ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
 Resource           ../../../Resources/Page/ODH/ODHDashboard/ODHDashboardSettings.resource
+Resource           ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProjects.resource
```

Comment:
what's this file? I think this should be removed because the correct one is `what's this file? I think can be removed`